include/pybind11/iostream.h

```diff
@@ -75,7 +75,7 @@ NAMESPACE_END(detail)
 
         ...
 
-        {
+        /* block */ {
```

Comment:
These shouldn't be necessary anymore (with the `awk` replacement I give below).

Reply:
Removed.